envsec/pkg/awsfed/awsfed.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/aws/aws-sdk-go-v2/service/cognitoidentity"
 	"github.com/aws/aws-sdk-go-v2/service/cognitoidentity/types"
+	"github.com/golang-jwt/jwt/v5"
```

Comment:
Use session.Token instead like existing functions.

Side note, could you make sure we're using `github.com/golang-jwt/jwt/v5` everywhere. I think we accidentally added v4s https://github.com/jetpack-io/axiom/blob/d4079463c5582b9906f5b8ba7b89ce39ee888218/pkg/jetdb/token.go#L6-L6